app/root/root.css

```diff
@@ -417,6 +417,14 @@ button {
   border-left: 6px solid #8bc34a;
 }
 
+.card.card-skipped {
+  border-left: 6px solid #888;
```

Comment:
Should we match the other cards and make the left border purple and leave the color of the target text as-is?